#159428325 Create the signin api connection to ui

kampkelly opened the pull request:
#### What does this PR do?
This pull request connects the sign in api endpoint to the signin page. This allows a user to be able to log into the app.

#### Description of Task to be completed?
The following tasks have been completed:
A user can sign in to the application.
The footer has been updated in all the html pages.
Scss files have been removed from .gitignore.

#### How should this be manually tested?
After cloning the repo, run npm install, then open signin.html from the FULLCONNECTION folder in any browser of your choice, preferably chrome.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#159428325:(https://www.pivotaltracker.com/n/projects/2183547)
#### Screenshots (if appropriate)
#### Questions:

kampkelly committed:
ft-create sign in api connection to ui-#159428325

This commit connects the sign in api endpoint to the signin page. This allows a user to be able to log into the app.
The footer has also been changed for all other files to be uniform.
Scss files have been removed from .gitignore.
The following files were added/changed:
- signin.html
- stylesheets/signup.css
- stylesheets/*.scss
- all html pages

Ft: #159428325

kampkelly committed:
ft-Change fetch api url to match heroku

This commit changes the fetch url to match the url of the production server on heroku.
The file following file was changed:
- javascripts/es6/script.js
- .gitignore

phemmz commented on FULLCONNECTION/javascripts/es6/script.js:
```diff
@@ -111,7 +111,7 @@ function submitSignup() {
 		} else {
 			document.querySelector('.form_error_text').style.display = 'none';
-			fetch('http://localhost:3000/api/v1/auth/signup', {
+			fetch('https://kampkelly-mydiary-api.herokuapp.com/api/v1/auth/signup', {
```
You can create a baseUrl ('https://kampkelly-mydiary-api.herokuapp.com/api/v1') and then append your route to it.

kampkelly committed:
ft-Change signin form buttons alignment-#159428325

This commit properly aligns the signin and signup buttons alignment. The main folder has also been renamed from ‘FULLCONNECTION’ to ‘Client’.

ft-#159428325
